Add install instructions for ArchLinux (#2153)
GenericMappingTools/gmt: INSTALL.md

include the supplements. If you need them you will need to
[build from source](BUILDING.md) or upgrade to 18.04 LTS.

### ArchLinux

It's easier to manage installed content and dependencies than directly
building and manually `make install` in ArchLinux.

    # Full update system packages first
    sudo pacman -Syu

    # Install tools for building AUR packages
    sudo pacman -S base-devel

>     # Use git command to clone Non-official AUR repo of gmt

Reviewer: ```suggestion
    # Use git command to clone non-official AUR repo of gmt
```

>     # here's two options, **you just have to clone one of them**

Reviewer: ```suggestion
    # Here are two options, **you just have to clone one of them**
```

>     # If you encounter some problems in those repo

Reviewer: ```suggestion
    # If you encounter some problems in those repo,
```

include the supplements. If you need them you will need to
[build from source](BUILDING.md) or upgrade to 18.04 LTS.

### ArchLinux

> 

Reviewer: ```suggestion
```

include the supplements. If you need them you will need to
[build from source](BUILDING.md) or upgrade to 18.04 LTS.

> ## ArchLinux

Reviewer: ```suggestion
### ArchLinux
```

> ### Install via building AUR package

Reviewer: ```suggestion
```

It's easier to manage installed content and dependencies than directly
building and manually `make install` in ArchLinux.

    # Full update system packages first
    sudo pacman -Syu

    # Install tools for building AUR packages
    sudo pacman -S base-devel

    # Use git command to clone Non-official AUR repo of gmt
    # here's two options, **you just have to clone one of them**
    # If you encounter some problems in those repo
    # you can report to the authors first, or choose another one to clone

> 

Reviewer: ```suggestion
```
